core/settings/base.py

```diff
@@ -144,5 +145,10 @@
 # Name of the URL where a user may login
 LOGIN_URL = 'auth_login'
 
-
 AUTH_USER_MODEL = 'register.User'
+
+OAUTH2_PROVIDER = {
```

Comment:
We had this already. The key thing is [here](https://github.com/etalab/id.data.gouv.fr/pull/21/files#diff-265524153427c5c3f0dd9ae087f16a4bR13), using the returned enriched request instead of the local request.